src/crypto/mode/gcm.js

```diff
@@ -36,12 +37,14 @@ const ALGO = 'AES-GCM';
 
 /**
  * Class to en/decrypt using GCM mode.
- * @param {String} cipher - The symmetric cipher algorithm to use e.g. 'aes128'
+ * @param {enums.symmetric} cipher - The symmetric cipher algorithm to use
  * @param {Uint8Array} key - The encryption key
  */
 async function GCM(cipher, key) {
-  if (cipher.substr(0, 3) !== 'aes') {
-    throw new Error('GCM mode supports only AES cipher');
+  if (cipher !== enums.symmetric.aes128 &&
+    cipher !== enums.symmetric.aes192 &&
+    cipher !== enums.symmetric.aes256) {
+    throw new Error('EAX mode supports only AES cipher');
```

Comment:
```suggestion
    throw new Error('GCM mode supports only AES cipher');
```
These error messages are quite broken English btw, but ok, that's a bit unrelated